src/lib/tls/tls_text_policy.cpp

```diff
@@ -15,6 +15,10 @@
 
 namespace Botan::TLS {
 
+bool Text_Policy::allow_ssl_key_log_file() const {
+   return get_bool("allow_ssl_key_log_file", Policy::allow_ssl_key_log_file());
+}
```

Comment:
Please also extend `Policy::print()` in `src/lib/tls/tls_policy.cpp` which is the serialization counterpart of this accessor method.